spec/features/vehicles/show_spec.rb

```diff
@@ -8,4 +8,22 @@
     expect(page).to have_content("Locked: true")
     expect(page).to have_content("Seats: 100")
   end
+
+  it 'Then I see a link to update the parent "Update Parent"' do
+    plane = Vehicle.create(name:"Boeing 747", locked: true, passenger_capacity: 100)
+    visit "vehicles/#{plane[:id]}"
+    expect(page).to have_link("Update Vehicle")
+    click_link "Update Vehicle"
+    expect(current_path).to eq("/vehicles/#{plane[:id]}/edit")
+    expect(page).to have_content("Update")
```

Comment:
Since the html has Update [vehicle name] why not put that in the test as well with:

`"Update #{plane[:name]}"`